cmd/frontend/main.go

```diff
@@ -65,6 +65,8 @@ var (
 
 	targetURLStr = flag.String("query.target-url", fmt.Sprintf("https://monitoring.googleapis.com/v1/projects/%s/location/global/prometheus", projectIDVar),
 		fmt.Sprintf("The URL to forward authenticated requests to. (%s is replaced with the --query.project-id flag.)", projectIDVar))
+
+	rulesEvaluatorURLStr = flag.String("rules.target-url", "http://rule-evaluator.gmp-system.svc.cluster.local:19092", "The URL to forward requests to the rules evaluator.")
```

Comment:
Question for future compatibility. Should we make it open ended and... accept list one day? (aka repeated string flag)

It's ~trivial to merge rules, we added same logic in Thanos Querier years ago (we could even import/copy the exact code): https://github.com/thanos-io/thanos/blob/main/pkg/rules/proxy.go

The reason is that truly scalable rule-eval is multi instance, so this will come handy sooner or later.

Reply:
@bwplotka 
That's a neat idea! I like it. Give us a day or three to change it, before we merge this :)